data.xml

```diff
@@ -13,7 +13,129 @@
                       category="user">
             <type placeholder=""/>
         </attributekey>
-
+        <attributekey handle="profile_private_messages_enabled" name="Private Messages" package="concrete_cms_theme"
```

Comment:
I don't think these attributekeys should be here. Shouldn't this have been moved into the community package?

Reply:
Okay i will move this to the community package.